ruby/whatsapp-with-vonage/lib/utils.rb

```diff
@@ -0,0 +1,23 @@
+def get_static_file(file_name)
+    parent_directory = File.expand_path('..', __dir__)
+
+    static_files_directory = File.join(parent_directory, 'static')
+
+    full_file_path = File.join(static_files_directory, file_name)
```

Comment:
Can this be simplified in any way?

Reply:
@loks0n okay, will work my way to simplified this
i'm sorry if my code is still messed in any way because i'm new to Ruby 😅